### PR TITLE
Added pages for all the crypto info half as placeholders and half wit…

### PR DESCRIPTION
**Pull request by robodrobo**
This will add pages where there were previously 404 errors, some placeholders but all to be updated. (By me)

### CODE REVIEW

**Comment by robodrobo** [comment]
@nzstartupdev Could you just take a look at this tomorrow to make sure I'm not going to break the handbook site...

File: app/(pages)/things-every-employee-should-know/bitcoin-basics/page.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+"use client";
+import { motion } from "framer-motion";
+import { Typography } from "@/components/ui/typography";
+import Link from "next/link";
+
+export default function GitBasics() {
```
Comment (nz-sus) [inline app/(pages)/things-every-employee-should-know/bitcoin-basics/page.tsx]: the default function name should correspond to the page name. "BitcoinBasics" in this case.

File: app/(pages)/things-every-employee-should-know/crypto-intro/page.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+"use client";
+import { motion } from "framer-motion";
+import { Typography } from "@/components/ui/typography";
+import Link from "next/link";
+
+export default function GitBasics() {
```
Comment (nz-sus) [inline app/(pages)/things-every-employee-should-know/crypto-intro/page.tsx]: the default function name should correspond to the page name. "CryptoIntro" in this case.

File: app/(pages)/things-every-employee-should-know/crypto-wallets/page.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+"use client";
+import { motion } from "framer-motion";
+import { Typography } from "@/components/ui/typography";
+import Link from "next/link";
+
+export default function GitBasics() {
```
Comment (nz-sus) [inline app/(pages)/things-every-employee-should-know/crypto-wallets/page.tsx]: the default function name should correspond to the page name. "CryptoWallets" in this case.

File: app/(pages)/things-every-employee-should-know/ethereum-overview/page.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+"use client";
+import { motion } from "framer-motion";
+import { Typography } from "@/components/ui/typography";
+import Link from "next/link";
+
+export default function GitBasics() {
```
Comment (nz-sus) [inline app/(pages)/things-every-employee-should-know/ethereum-overview/page.tsx]: the default function name should correspond to the page name. "EthereumOverview" in this case.

File: app/(pages)/things-every-employee-should-know/bitcoin-basics/page.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+            exit={{ opacity: 0, y: -20 }}
+            transition={{ duration: 0.5 }}
+            className="container mx-auto px-4 py-8"
+        >
+            <Typography variant="h1" className="mb-8 text-4xl border-b border-red-500 pb-2">
+                Bitcoin Basics
+            </Typography>
+            <Typography variant="body" className="mb-4">
+                Bitcoin is kind of a strange thing for people to grasp, mostly because you can’t physically grasp it, but it can still be used to purchase goods and services.
+
```
Comment (nz-sus) [inline app/(pages)/things-every-employee-should-know/bitcoin-basics/page.tsx]: Newlines and whitespace are not respected by HTML tags. Use separate components or a break tag <br/> for spacing.

File: app/(pages)/things-every-employee-should-know/crypto-wallets/page.tsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+            exit={{ opacity: 0, y: -20 }}
+            transition={{ duration: 0.5 }}
+            className="container mx-auto px-4 py-8"
+        >
+            <Typography variant="h1" className="mb-8 text-4xl border-b border-red-500 pb-2">
+                Crypto Wallets
+            </Typography>
+            <Typography variant="body" className="mb-4">
+                There are lots of options when it comes to what type of wallet you can use. For the sake of simplicity, I’m going to suggest you get started with Bitcoin core (https://bitcoin.org/en/download). This is an application that you install and will set up a wallet for you. Two things to note: 1. This will take up A LOT of space (around 40gigs at the time that I write this) since it syncs the whole blockchain to your hard drive and 2. Your wallet will be a file on your hard drive so if your hard drive dies your Bitcoin is all lost.
+
```
Comment (nz-sus) [inline app/(pages)/things-every-employee-should-know/crypto-wallets/page.tsx]: Newlines and whitespace are not respected by HTML tags. Use separate components or a break tag <br/> for spacing.

**Review by nz-sus — COMMENTED** [review]
Good start! For future branches, please use a personal identifier so it's easier to correlate branches to owners. I'm moving to use noahz/ myself.

File: app/(pages)/things-every-employee-should-know/bitcoin-basics/page.tsx (hunk 5 above)
Comment (robodrobo) [reply app/(pages)/things-every-employee-should-know/bitcoin-basics/page.tsx]: added <br></br> to lines, thanks

File: app/(pages)/things-every-employee-should-know/bitcoin-basics/page.tsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+            animate={{ opacity: 1, y: 0 }}
+            exit={{ opacity: 0, y: -20 }}
+            transition={{ duration: 0.5 }}
+            className="container mx-auto px-4 py-8"
+        >
+            <Typography variant="h1" className="mb-8 text-4xl border-b border-red-500 pb-2">
+                Bitcoin Basics
+            </Typography>
+            <Typography variant="body" className="mb-4">
+                <br>Bitcoin is kind of a strange thing for people to grasp, mostly because you can’t physically grasp it, but it can still be used to purchase goods and services.</br>
```
Comment (nz-sus) [inline app/(pages)/things-every-employee-should-know/bitcoin-basics/page.tsx]: br tags are self contained line breaks (I prefer to use "br/") and do not have a closing /br

**Review by nz-sus — CHANGES_REQUESTED** [review]
remove all of the ending `</br>`s and add an ending / to br it should work

File: app/(pages)/things-every-employee-should-know/bitcoin-basics/page.tsx (hunk 7 above)
Comment (robodrobo) [reply app/(pages)/things-every-employee-should-know/bitcoin-basics/page.tsx]: That's what I thought I only ever put breaks at the end of a line, let me try that. Thanks again.

**Review by nz-sus — APPROVED** [review]
LGTM

**Pull request merged**